packages/circuit-ui/components/Popover/Popover.tsx

```diff
@@ -356,6 +356,7 @@ export const Popover = ({
     triggerEl.current,
     popperElement,
     {
+      strategy: 'fixed',
```

Comment:
The popovers used to be absolutely positioned which caused issues when resizing the screen to a smaller width. Since the popovers are not destroyed they ended up off-screen, stretching the main app container and causing the side panel to behave in a weird manner, leading to a gap between it and the main content and also pushing the content below the side nav. This also fixes a content sizing issue when using the responsive mode from chrome dev tools.
<img width="934" alt="Screenshot 2022-02-09 at 13 49 18" src="https://user-images.githubusercontent.com/9676303/153194272-555c7ca7-a9b9-41b6-8556-6acb5f2dcecb.png">